### PR TITLE
Java refactoring

### PR DESCRIPTION
**Pull request by andryspo**

### CODE REVIEW

File: src/main/java/ua/softserveinc/tc/dto/BookingDto.java [hunk 1]
```diff
@@ -315,14 +317,6 @@ public void setId(Long id) {
         this.id = id;
     }
 
-    public Long getDurtionLong() {
```
Comment (hrybs) [inline src/main/java/ua/softserveinc/tc/dto/BookingDto.java]: Why have you delete this?

File: src/main/webapp/resources/js/day-off-service.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-App.factory('DayOffService', ['$http', '$q', function ($http, $q) {
```
Comment (hrybs) [inline src/main/webapp/resources/js/day-off-service.js]: Is it save delete this?

File: src/main/webapp/resources/js/edit-my-kid.js [hunk 3]
```diff
@@ -23,20 +23,20 @@ $(function () {
...
     $.post('remove-kid/' + kidId)
-        .success(function (responseText) {
+        .success(function () {
             $(location).attr('href', kidredirect).reload(true);
         })
         .error(function (xhr, status, error) {
-            console.log("Error!" + error + "  status = " + status);
+            console.log('Error!' + error + ' status = ' + status);
         });
 }
```
Comment (hrybs) [inline src/main/webapp/resources/js/edit-my-kid.js]: No new line.

File: src/main/webapp/resources/js/for-rates.js [hunk 4]
```diff
@@ -3,30 +3,28 @@ var app = angular.module('angularjs-starter', []);
...
+        $.ajax({
+            type: 'post',
+            contentType: 'application/json',
+            url: 'get-json',
+            dataType: 'text',
+            processData: false,
+            data: JSON.stringify($scope.choices)
+        });
+    };
 });
```
Comment (hrybs) [inline src/main/webapp/resources/js/for-rates.js]: No new line.

File: src/main/webapp/resources/js/header-manager.js [hunk 5]
```diff
@@ -1,40 +1,40 @@
...
 function getAmountOfChildren() {
     $.ajax({
-        url: "getAmountOfChildren/" + localStorage["roomId"],
+        url: 'getAmountOfChildren/' + localStorage['roomId'],
         success: function (result) {
-            $("#amountOfChildren").text(result)
+            $('#amountOfChildren').text(result);
         }
     });
 }
```
Comment (hrybs) [inline src/main/webapp/resources/js/header-manager.js]: No new line.

File: src/main/webapp/resources/js/header-user.js [hunk 6]
```diff
@@ -26,21 +26,19 @@ function getManagersNames(arr) {
...
-
-            selectRoomForUser(localStorage["userRoomId"], localStorage["userId"], localStorage["phoneNumber"], localStorage["managers"]);
-            $("#usersRoom").text(localStorage["address"]);
+            selectRoomForUser(localStorage['userRoomId'], localStorage['userId'], localStorage['phoneNumber'], localStorage['managers']);
+            $('#usersRoom').text(localStorage['address']);
 
         });
-
     });
 }
```
Comment (hrybs) [inline src/main/webapp/resources/js/header-user.js]: No new line.

File: src/main/java/ua/softserveinc/tc/dto/MonthlyEventDto.java [hunk 7]
```diff
@@ -17,24 +17,26 @@
...
-        super(event);
-    }
-
-    public void setDaysOfMonth(int[] daysOfTheMonth) {
+    public void setDaysOfTheMonth(int[] daysOfTheMonth) {
         this.daysOfTheMonth = daysOfTheMonth;
     }
 
-    public int[] getDaysOfMonth() {
+    public int[] getDaysOfTheMonth() {
```
Comment (Pitcity) [inline src/main/java/ua/softserveinc/tc/dto/MonthlyEventDto.java]: be carefull with merging this one pls, i've made some changes, need it for demo))

**Pull request merged**